backend/api/views/model.py

```diff
@@ -25,6 +34,29 @@
 logger = structlog.get_logger(__name__)
 
 
+def create(request, basename, get_success_headers):
```

Comment:
```suggestion
def _create(request, basename, get_success_headers):
```

This should probably not be used elsewhere.